packages/nextjs/components/MetaHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Head from "next/head";
+
+interface MetaHeaderProps {
+  title?: string;
+  description?: string;
+  favicon?: string;
+  ogTitle?: string;
+  ogDescription?: string;
+  ogImage?: string;
+  twitterCard?: string;
+  twitterTitle?: string;
+  twitterDescription?: string;
+  twitterImage?: string;
+}
+
+const MetaHeader: React.FC<MetaHeaderProps> = ({
+  title,
```

Comment:
I think `title / ogTitle / twitterTitle` should be the same, so we only need one prop "title".

same for description and image